pages/account.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "../styles/pages/account.module.scss";
+import AccountUserName from "../components/account/AccountUserName";
+
+export default function Account() {
+	const daysAlive = 9;
+	const [profilePicSrc, setProfilePicSrc] = useState("/lio_img.png");
+
+	const onUpload = (event: React.ChangeEvent<HTMLInputElement>) => {
+		try{
+			if(event.target.files){
+			const file = event.target.files[0];
+			setProfilePicSrc(URL.createObjectURL(file));
+		}else{
+			console.log("no files detected");
+		}}
+		catch {
```

Comment:
I still think this try/catch is not good and if an error does rise, then there's problem that needs to be fixed.
This isn't universal, a lot of times you'd need/want the try/catch, just don't think here is the case.

Reply:
noted